packages/manager/apps/telecom/src/app/telecom/pack/migration/building-details/migration-building-details.html

```diff
@@ -66,31 +66,63 @@
+                            name="radioPto"
+                            model="$ctrl.model.pto"
+                            value="'no'"
+                        >
+                            <span
+                                data-translate="telecom_pack_migration_pto_check_no"
+                            ></span>
+                            <button
+                                type="button"
+                                class="oui-popover-button"
+                                oui-popover="{{'telecom_pack_migration_pto_no_description' | translate}}"
+                            ></button>
+                        </oui-radio>
+                    </div>
                 </div>
-                <div class="form-group">
-                    <oui-radio
-                        name="radioPto"
-                        model="$ctrl.model.pto"
-                        value="false"
-                    >
-                        <span
-                            data-translate="telecom_pack_migration_pto_check_no"
-                        ></span>
-                    </oui-radio>
+                <div class="form-inline">
+                    <div class="form-group">
+                        <oui-radio
+                            name="radioPto"
+                            model="$ctrl.model.pto"
+                            value="'yesNotKnown'"
+                        >
+                            <span
+                                data-translate="telecom_pack_migration_pto_yes_not_known"
+                            ></span>
+                            <button
+                                type="button"
+                                class="oui-popover-button"
+                                oui-popover="{{'telecom_pack_migration_pto_yes_not_known_description' | translate}}"
+                            ></button>
+                        </oui-radio>
+                    </div>
+                </div>
+                <div class="form-inline">
+                    <div class="form-group">
+                        <oui-radio
+                            name="radioPto"
+                            model="$ctrl.model.pto"
+                            value="'yes'"
+                        >
+                            <span
+                                data-translate="telecom_pack_migration_pto_yes"
+                            ></span>
+                            <button
+                                type="button"
+                                class="oui-popover-button"
+                                oui-popover="{{'telecom_pack_migration_pto_yes_description' | translate}}"
+                            ></button>
+                        </oui-radio>
+                    </div>
                 </div>
             </div>
-            <div class="form-inline" data-ng-if="$ctrl.model.pto">
+            <div class="form-inline" data-ng-if="$ctrl.model.pto === 'yes'">
```

Comment:
Could this be at least in a function something like 
```suggestion
            <div class="form-inline" data-ng-if="$ctrl.isPto()">
```